pype/plugins/blender/load/load_animation.py

```diff
@@ -0,0 +1,256 @@
+"""Load an animation in Blender."""
+
+import logging
+from pathlib import Path
+from pprint import pformat
+from typing import Dict, List, Optional
+
+from avalon import api, blender
+import bpy
+import pype.blender.plugin
+
+
+logger = logging.getLogger("pype").getChild(
+    "blender").getChild("load_animation")
+
+
+class BlendAnimationLoader(pype.blender.plugin.AssetLoader):
+    """Load animations from a .blend file.
+
+    Warning:
+        Loading the same asset more then once is not properly supported at the
+        moment.
+    """
+
+    families = ["animation"]
+    representations = ["blend"]
+
+    label = "Link Animation"
+    icon = "code-fork"
+    color = "orange"
+
+    @staticmethod
+    def _remove(self, objects, lib_container):
```

Comment:
I am not sure why those methods in Loader plugin are market as static methods. I couldn't find place where they are statically called and when they are, `self` is passed manually.

Reply:
You are right, I'm not sure why I set them as static! I'll change that.